runtime/src/main/java/org/corfudb/runtime/view/AddressSpaceView.java

```diff
@@ -56,6 +55,18 @@
 @Slf4j
 public class AddressSpaceView extends AbstractView {
 
+    public static long removalCauseCacheSize = 500_000;
+
+    /**
+     * This cache maintains debug information that is required to
+     * determine the cause of a NoRollbackException. Using ehcache's
```

Comment:
typo: Using the* cache's deepSizeOf shows that ..

Reply:
ehcache is the library the provides deepSizeOf :)

Reply:
sorry newbie mistake :)